### PR TITLE
feat: add contact form

### PR DESCRIPTION
**Pull request by rifkiandriyanto**

### CODE REVIEW

File: src/ContactForm.tsx [hunk 1]
```diff
@@ -2,74 +2,87 @@ import React, { useState } from "react";
...
+            <label
+              htmlFor="email"
+              className="block mb-2 text-sm font-medium text-gray-900"
+            >
+              Your email
+            </label>
             <input
-              value={message}
-              onChange={(e) => setMessage(e.target.value)}
+              value={eamil}
```
Comment (stefandavid-ajaib) [inline src/ContactForm.tsx]: typo nih

File: src/templates/Banner.tsx [hunk 2]
```diff
@@ -4,10 +4,8 @@ import { Section } from "../layout/Section";
 const Banner = () => (
   <Section top>
     <CTABanner
-      title="
-      Tingkatkan Gaya Hidup Aktifmu Bersama Kami! "
-      subtitle="
-      Daftar dalam waiting list dan dapatkan update saat kami sudah rilis"
+      title="Get In Touch"
+      subtitle="Tingkatkan gaya hidup aktifmu bersama kami!"
```
Comment (stefandavid-ajaib) [inline src/templates/Banner.tsx]: why is this not referring to the copy provided on the figma?

File: src/ContactForm.tsx [hunk 3]
```diff
@@ -2,74 +2,87 @@ import React, { useState } from "react";
...
-              Email
+              Subject
             </label>
+            <input
+              value={subject}
+              onChange={(e) => setSubject(e.target.value)}
+              type="text"
+              id="subject"
+              className="block p-3 w-full text-sm text-gray-900 bg-gray-50 rounded-lg border border-gray-300 shadow-sm focus:ring-primary-500 focus:border-primary-500"
+              placeholder="Let us know how we can help you"
```
Comment (stefandavid-ajaib) [inline src/ContactForm.tsx]: we can make this in indo please. "Tinggalkan pesan.."

File: src/ContactForm.tsx [hunk 4]
```diff
@@ -2,74 +2,87 @@ import React, { useState } from "react";
...
+            >
+              Your Note
+            </label>
+            <textarea
+              value={message}
+              onChange={(e) => setMessage(e.target.value)}
+              id="message"
+              rows={6}
+              className="block p-2.5 w-full text-sm text-gray-900 bg-gray-50 rounded-lg shadow-sm border border-gray-300 focus:ring-primary-500 focus:border-primary-500"
+              placeholder="Leave a comment..."
```
Comment (stefandavid-ajaib) [inline src/ContactForm.tsx]: will this be visible before typing? if yes, we should make this in indo please

File: src/ContactForm.tsx [hunk 5]
```diff
@@ -2,74 +2,87 @@ import React, { useState } from "react";
...
+  const [eamil, setEmail] = useState("");
+  const [subject, setSubject] = useState("");
   const [message, setMessage] = useState("");
 
   const onSubmit = async (e: any) => {
     e.preventDefault();
     setSubmitted(true);
-    await submit({ message });
+    await submit({ eamil, subject, message });
   };
```
Comment (stefandavid-ajaib) [inline src/ContactForm.tsx]: typo on the "eamil"

**Comment by rickithadi** [comment]
logos in the footer with corresponding links need updating

Screenshot 2023-08-17 at 4 47 15 PM

**Review by stefandavid-ajaib — APPROVED** [review]

**Pull request merged**